jasmine/spec/inverted-index-test.js

```diff
@@ -18,30 +18,36 @@ describe('Read book data', () => {
   });
   it('parses received files into JSON objects', () => {
     expect(typeof (invertedIndexTest.validFiles(wrongFile))).toBe(typeof {});
-  }); 
+  });
+  it('reads a JSON file and asserts that it valid', () => {
+    expect(invertedIndexTest.validFiles(JSON.stringify(validBook))[1]).toBe('Success');
+  });
+  it('reads a JSON file and asserts that it valid', () => {
+    expect(invertedIndexTest.validFiles(JSON.stringify(trollTest))[1]).toBe('this Index takes books with Title and Text property only');
+  });
   it('reads a JSON file and asserts that it valid', () => {
-     expect(invertedIndexTest.validFiles(JSON.stringify(validBook))[1]).toBe('Success');
+    expect(invertedIndexTest.validFiles(JSON.stringify(wrongFile))[1]).toBe('this Index takes books with Title and Text property only');
```

Comment:
You might want to disable the no-undef rule for this file
you can put `/*eslint no-undef:0*/` at the top of this file